docs/deployment/feature_gates.md

```diff
@@ -208,3 +209,4 @@ A *General Availability* (GA) feature is also referred to as a *stable* feature.
 | ShootManagedIssuer              | `gardenlet`                       | Enables the shoot managed issuer functionality described in GEP 24.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
 | VPAForETCD                      | `gardenlet`, `gardener-operator`  | Enables VPA for `etcd-main` and `etcd-events`, regardless of HVPA enablement.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
 | VPAAndHPAForAPIServer           | `gardenlet`, `gardener-operator`  | Enables an autoscaling mechanism for `kube-apiserver` of shoot or virtual garden clusters, and the `gardener-apiserver`. They are scaled simultaneously by VPA and HPA on the same metric (CPU and memory usage). The pod-trashing cycle between VPA and HPA scaling on the same metric is avoided by configuring the HPA to scale on average usage (not on average utilization) and by picking the target average utilization values in sync with VPA's allowed maximums. The feature gate takes precedence over the `HVPA` feature gate when they are both enabled. |
+| WorkerPoolHashWithNodeAgentSecret | `gardenlet`                       | Enables usage of the new worker pool hash calculation. All provider extension must be upgraded to support this feature first.enablement.                                                                                                                                                                                                                                                                                                                                                                                                                            |
```

Comment:
Should we not include more functional details why this might be interesting/what this means?
The most notable change will be that changes to `kubeReserved` will result in a node rollout. Apart from that, it should not have notable changes as far as I understand.

Reply:
I've added a remark regarding kubeResered, evictionHard and cpuManagerPolicy